Dockerfile

```diff
@@ -97,8 +100,6 @@ RUN dpkg --add-architecture i386 \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 COPY supervisord.conf /etc/supervisor/supervisord.conf
 
-ENV TZ=Etc/UTC
```

Comment:
```
lukas@MACROMI:~$ docker run --rm -it debian:stable-slim /bin/bash
root@390ed9c45055:/# echo $TZ

root@390ed9c45055:/#
```

Reply:
Without this supervisor can not query the TZ env var unless the user has explicitly specified it.
https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html

All the scripts are fine as they will get it from `defaults` but supervisor isn't.

Reply:
Actually, now that we have the new `bootstrap` script and supervisord is no longer the entrypoint we could get it from `defaults` before the supervisor `exec`. I'm fine with either.

Reply:
@lloesche Yeah I think it's best to not define that here in the Dockerfile.  How would we add it to the `bootstrap` script?  Would `export TZ="$TZ"` before the `exec` work?  (This is a bit out of my wheelhouse 😛)

Reply:
Just `export TZ` right before the exec is enough since the value is already imported on line 5.